Update files to follow latest amazon linux changes

ng-htec opened the pull request:
update: init.yaml
update: variables.tf. instance type changed, cpu limit changed

Commit:
update: init.yaml
update: variables.tf. instance type changed, cpu limit changed

jzlatanovic commented on variables.tf:
```diff
@@ -35,7 +35,7 @@ variable "subnet_ids" {
 variable "instance_type" {
   type        = string
-  default     = "t3.nano"
+  default     = "t2.micro"
```
Set t3 instance type as default, avoid older instance types such as t2

jzlatanovic reviewed (commented):
Please check the TFSec findings

Commit:
update: data.tf aws_caller_identity added
update: iam.rf restricted resourse access
update: variables.tf instance type changed

ng-htec commented:
> Please check the TFSec findings

Inbound from outside is necessary but they are able to modify it and allow access from only certain ip/subnet

Resource issue has been updated and limited

Commit:
update: main.t.f, set metadatadata options for launch template

Commit:
update: main.tf, description for security group resource updated

Commit:
update: variables.tf add ingress_cidr_blocks, ingress_ipv6_cidr_blocks, egress_cidr_blocks variables
update: main.tf, aws_security_group resource updated

Commit:
update: main.tf, modify aws_security_group resource and add ignore for tfsec
update: variables.tf, add wireguard_ingress_settings var

Commit:
update: main.tf, resource aws_launch_template metadata_options updated

jzlatanovic reviewed (approved):
Splendid 😄

Commit:
update: description for aws_security_group ingress
update: variables.tf, description for wireguard_ingress_settings var updated
